sdk/tables/Azure.Data.Tables/tests/perf/Infrastructure/EntityGenerator.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
```

Comment:
nit:  You may want to add a blank line for consistency with other files.

Reply:
After looking the PR over, I'm not sure this is used since the base class has a generate method....